pom.xml

```diff
@@ -178,6 +177,8 @@
     <process-exec-maven-plugin.version>0.8</process-exec-maven-plugin.version>
     <swagger-maven-plugin.version>1.0.0</swagger-maven-plugin.version>
     <swagger2markup-maven-plugin.version>1.1.0</swagger2markup-maven-plugin.version>
+    <sulky-.version>8.1.1</sulky-.version>
```

Comment:
I don't see this being used, am I missing something?

Reply:
Removing